src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -197,12 +197,48 @@ void GraphingCalculator::OnEquationsVectorChanged(IObservableVector<EquationView
     GraphingControl->PlotGraph(false);
 }
 
+wstringstream GraphingCalculator::FormatTraceValue(double min, double max, float pointValue)
+{
+    wstringstream traceValueString;
+    auto roundingNumber = pow(10, floor(log10(max - min)) - 3);
```

Comment:
We should add a comment explaining what we try to achieve here.

Reply:
We should extract the precision:
`auto precision = floor(log10(...));`

And use it line 224

Reply:
Good idea